app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,2 @@
+class PublicRecipesController < ApplicationController
+end
```

Comment:
No methods are defined for this class. I think this may me the source of the error

Reply:
Ok I added a method for the public_recipes_controller, according your suggestions. But, according the screen capture you sent, it seems that is because a missing method for public_recipes in the Recipes Controller.